test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -220,56 +219,37 @@ var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", fun
 	})
 
 	It("should scale down underutilized nodes [Feature:ClusterAutoscalerScalability4]", func() {
-		underutilizedReservation := int64(float64(memCapacityMb) * 0.01)
-		fullReservation := int64(float64(memCapacityMb) * 0.8)
-		perNodeReplicas := 10
+		perPodReservation := int(float64(memCapacityMb) * 0.01)
+		// underutilizedNodes are 10% full
+		underutilizedPerNodeReplicas := 10
+		// fullNodes are 70% full
+		fullPerNodeReplicas := 70
 		totalNodes := maxNodes
+		underutilizedRatio := 0.3
 
 		// resize cluster to totalNodes
 		newSizes := map[string]int{
 			anyKey(originalSizes): totalNodes,
 		}
 		setMigSizes(newSizes)
-		framework.ExpectNoError(framework.WaitForClusterSize(f.ClientSet, totalNodes, largeResizeTimeout))
-
-		// annotate all nodes with no-scale-down
-		ScaleDownDisabledKey := "cluster-autoscaler.kubernetes.io/scale-down-disabled"
-
-		nodes, err := f.ClientSet.Core().Nodes().List(metav1.ListOptions{
-			FieldSelector: fields.Set{
-				"spec.unschedulable": "false",
-			}.AsSelector().String(),
-		})
-
-		framework.ExpectNoError(addAnnotation(f, nodes.Items, ScaleDownDisabledKey, "true"))
-
-		// distribute pods (using taints)
-		divider := int(float64(len(nodes.Items)) * 0.7)
 
-		fullNodes := nodes.Items[:divider]
-		underutilizedNodes := nodes.Items[divider:]
+		framework.ExpectNoError(framework.WaitForClusterSize(f.ClientSet, totalNodes, largeResizeTimeout))
```

Comment:
Large resize timeout is basically just enough time for CA to start scaling-down nodes under us. Aren't we worried about that?